mopidy_beetslocal/library.py

```diff
@@ -495,7 +495,7 @@ def _convert_item(self, item):
             track_kwargs['bitrate'] = item['bitrate']
 
         if 'mtime' in item:
-            track_kwargs['last_modified'] = item['mtime']
+            track_kwargs['last_modified'] = long(item['mtime'])
```

Comment:
In Mopidy 1.0, the semantics of the `Track.last_modified` field was changed from "increasing counter and seconds from epoch" to "increasing counter and milliseconds from epoch". This change was made to match the semantics of `Playlist.last_modified`. Ref https://docs.mopidy.com/en/latest/changelog/#id3.

Also, as the `long()` type is gone in Python 3, it is probably better to keep to `int()`, which also supports making longs when needed on Python 2.7.

``` python
import os, sys
>>> mtime = os.stat('/home/jodal').st_mtime
>>> mtime
1440342189.6026597
>>> mtime * 1000
1440342189602.6597
>>> int(mtime * 1000)  # on a 64-bit system
1440342189602
>>> int(mtime * 1000)  # on a 32-bit system
1440342189602L
```

Thus, a better more complete and future proof fix is probably:

``` python
track_kwargs['last_modified'] = int(item['mtime'] * 1000)
```